README.md

```diff
@@ -173,7 +175,7 @@ To use Earthdata Login with a locally running Harmony, you must first set up a n
 3. Add the necessary Required Application Group
 4. Update .env with credentials
 
-You must select "401" as the application type for Harmony to work correctly with command line clients and clients like QGIS.  You will also need to add the "eosdis_enterprise" group to the list of required application groups in order for CMR searches issued by Harmony to be able to use your Earthdata Login tokens.  Update `OAUTH_CLIENT_ID` and `OAUTH_PASSWORD` in .env with the information from your Earthdata Login application. Additional information including other OAUTH values to use when creating the application can be found in the example/dotenv file in this repository.
+You must select "401" as the application type for Harmony to work correctly with command line clients and clients like QGIS. Set the redirect URL to http://localhost:3000/oauth2/redirect for local Harmony. Leave Required User information empty. Redirect Time for Earthdata Login Splash page (in seconds): 3 seconds. Check the checkbox for By checking this box, I confirm that my application is compatible with EDL policy. Leave the other checkbox unchecked. Then create the new application. After the application is created, you can use the "manage" -> "App Groups" tab to add the "EOSDIS Enterprise" group to the application. This "EOSDIS Enterprise" group will allow CMR searches issued by Harmony to be able to use your Earthdata Login tokens.  Update `OAUTH_CLIENT_ID`, `OAUTH_UID` and `OAUTH_PASSWORD` in .env with the information from your Earthdata Login application. 
```

Comment:
I despise the Earthdata Login Splash and would never recommend setting it to anything above 0. There's no reason for artificial sleeps. I would at least remove the sentence from our documentation "Redirect Time for Earthdata Login Splash page (in seconds): 3 seconds."